packages/lesswrong/components/posts/ElicitBlock.tsx

```diff
@@ -63,6 +63,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   histogramBucket: {
     display: 'flex',
     flexGrow: 1,
+    overflow: 'hidden',
```

Comment:
This causes the percentage overlay to get hidden/chopped in some contexts.
![Screen Shot 2020-11-21 at 14 19 29](https://user-images.githubusercontent.com/101191/99888848-9f5b8a80-2c04-11eb-8253-9c158c486856.png)

Reply:
Oops, sorry, the whole point of this was to remove this, because it was indeed causing problems.